NuGet.Docs/ndocs/Policies/Dispute-Resolution.md

```diff
@@ -1,49 +1,16 @@
+In this scenario, Nancy does not appear to be acting with bad intentions, but is rather supporting Northwind's tools and customers by contributing her own time and code. At the same time, Northwind is the legitimate owner of the Northwind name.
 
-•	Attempting to make the gallery do something that it is not explicitly designed to do.
+By following the process below, Northwind and Nancy can work together to a suitable resolution, because both are interested in serving the developer community. It's typically not necessary for the NuGet team to become involved; collaboration usually works out best. In fact, every dispute brought to the NuGet team's attention to date has been worked out without the team needing to pass judgment.
 
 
-If you find a package that is in violation of any of these items, click the ‘Report Abuse’ link on the package details page and submit a report.
-
-
-## Summary
-
-The growth of the NuGet ecosystem has been amazing.  There are a lot of packages out there now, and more software companies are getting involved and publishing packages.  This sometimes brings about disputes similar to the following scenarios (fictitious names used):
-
-1) Northwind Traders makes a CRM system and they provide client drivers as a downloadable MSI from their website.  Alan wants to make it easier to use Northwind’s client library, and turns it into a NuGet package called “NorthwindTraders.Client”  Northwind would like to maintain an official NuGet package for their client library, and files a dispute against Alan.
-
-2) Nancy is a customer of the AdventureWorks web service and wrote a library to make programmatic usage of their service easier.  She bundled this library as a NuGet package and published it as “Nancy.AdventureWorks.WebClient”  AdventureWorks now has an official client library available and would like to direct developers to that library instead of the tool that Nancy previously published.
-
-In both scenarios Alan and Nancy do not appear to be acting with bad intentions, rather they are supporting these tools by contributing their own time and code to make improvements and help other customers of Northwind and AdventureWorks. 
-
-To date, every dispute brought to NuGet's attention has been resolved between the individual parties without NuGet administrators needing to get involved.  Many of these disputes have been resolved between organizations and developers because they share the same goal: make a developers job easier.
-
-The NuGet ecosystem is moving at a fast moving pace, and everyone wants their projects to “just work”.  When people collaborate to find the best solution, it usually works out much better than if a third party gets involved and makes a judgement.  We know there are very good lawyers at many organizations but it is our belief that when developers communicate with other developers a better technical solution for the community can be achieved.
-
-## Changes
+## Process
 
```

Comment:
It's essentially in the fourth paragraph of the edits, just greatly simplified. Are there parts that you think are still essential? The collaboration concept is there, as is the idea that the NuGet team probably doesn't need to be involved. The part about the ecosystem moving at a fast pace would have to be qualified with proof, and it's not necessary to the point of the policy. The part about lawyers seems unnecessary.